packages/graphql/schemas/schema.graphql

```diff
@@ -760,6 +760,7 @@ type GitInfo {
 enum GitInfoStatusType {
   created
   modified
+  no_git_info
```

Comment:
Not seeing the related `gql-` enum change, was this edited in the SDL file directly?

Also, we generally use camelCase everywhere for variable naming, unless it's `CONSTANT_CASE`, so `noGitInfo`

Reply:
I corrected to follow convention.
The main update is in `packages/types/src/git.ts`
https://github.com/cypress-io/cypress/blob/b8884b58f1d4b4fa5db90cb7867dcd593c370727/packages/types/src/git.ts#L1